src/main/resources/swagger.api/circulation-item.yaml

```diff
@@ -165,6 +180,34 @@ components:
     uuid:
       $ref: schemas/uuid.json
   parameters:
+    cqlQueryCirculationItem:
+      name: query
+      in: query
+      schema:
+        type: string
+        default: cql.allRecords=1
+      required: false
+      description: A query expressed as a CQL string
+    offset:
+      name: offset
+      in: query
+      schema:
+        type: integer
+        default: 0
+        minimum: 0
+        maximum: 2147483647
+      required: false
+      description: Skip over a number of elements by specifying an offset value for the query
+    limit:
+      name: limit
+      in: query
+      schema:
+        type: integer
+        default: 1000
+        minimum: 1
+        maximum: 2147483647
```

Comment:
Is the minimum and maximum value mentioned here are the standard values for FOLIO? I mean how these values get derived from, should we check with Kalibek about it ?

Reply:
@gurleenkaurbp :- Yes these are standard values
https://github.com/search?q=org%3Afolio-org%202147483647&type=code#:~:text=2140-,maximum%3A%202147483647,-2141

Reply:
Thanks @SinghAdes